index.html

```diff
@@ -41,34 +41,39 @@ <h1>Codified Conduct</h1>
         el: '#app',
         data: {
           items: [
+            {
+              title: 'Open Source Code of Conduct Guide',
+              url: 'https://opensource.guide/code-of-conduct/',
+              description: 'Here is a guide on using a code of conduct for your open source projects.'
+            },
             {
               title: '"No Ideologies"',
               url: 'https://github.com/CodifiedConduct/coc-no-ideologies',
-              description: ''
+              description: 'A code of conduct focused on the restriction of discussion to technical topics, and disallowing discussions pertaining to non-technical ideologies.'
             },
             {
               title: 'Django',
               url: 'https://www.djangoproject.com/conduct',
-              description: ''
+              description: 'This code of conduct has six basic commandments: "Be friendly and patient", "Be welcoming", "Be considerate", "Be respectful", "Be careful in the words you choose", and "Try to understand disagreements".'
             },
             {
               title: 'Citizen',
-              url: 'http://citizencodeofconduct.org',
-              description: ''
+              url: 'https://web.archive.org/web/20200330154000/http://citizencodeofconduct.org/',
+              description: 'This code of conduct outlines acceptable behaviour, unacceptable behavior and the consequences for unnacceptable behavior.'
             },
             {
               title: 'JavaScript Foundation',
-              url: 'https://js.foundation/community/code-of-conduct',
-              description: ''
+              url: 'https://github.com/openjs-foundation/cross-project-council/blob/main/CODE_OF_CONDUCT.md',
+              description: 'The summary of this CoC is to be a decent person and anything offensive that would not be tolerated at a workplace environment will not be tolerated in the community. '
             },
             {
               title: 'Contributor Covenant',
               url: 'https://www.contributor-covenant.org',
-              description: ''
+              description: 'A healthy open source community centers the shared values and norms of its members. While not all of these values are the exactly the same, there is a set of core values and norms that are essential in a just and equitable software commons. Contributor Covenant is a code of conduct that you can adapt to express both fundamental shared values, and the special norms and values that distinguish your own community.'
```

Comment:
we can loop back around to this one later
```suggestion
              description: ''
```